hw/dv/tools/ralgen/ralgen.py

```diff
@@ -60,11 +60,11 @@ def main():
     if ip_hjson:
         ral_spec = get_full_path(root_dir, ip_hjson)
         cmd = os.path.join(util_path, "regtool.py")
-        args = [cmd, "-s", "-t", ".", ral_spec]
+        args = [cmd, "--nowarn", "-s", "-t", ".", ral_spec]
     else:
         ral_spec = get_full_path(root_dir, top_hjson)
         cmd = os.path.join(util_path, "topgen.py")
-        args = [cmd, "-r", "-o", ".", "-t", ral_spec]
+        args = [cmd, "--nowarn", "-r", "-o", ".", "-t", ral_spec]
```

Comment:
@sriyerg @cindychip 

this may not be the cleanest solution, but it does the trick.
let me know if this works for you or if you have a better idea.

Reply:
I really dislike this. Let's instead try to get a warning-free build. I'm going to drop this patch from this PR (since it's got my name on it, I reserve the right to behave like a despot :-D), but will push it up as a draft PR so that it doesn't get lost. I'd prefer not to do this, though.

Reply:
I didn't claim it was a pretty solution at all ;). 
we can also try to clean up the warning messages in topgen as much as possible and find a way to waive the remaining ones.